pyart/graph/gridmapdisplay.py

```diff
@@ -74,14 +74,14 @@ def __init__(self, grid, debug=False):
         self.debug = debug
 
         # set up the projection
-        lat0 = grid.axes['lat']['data'][0]
-        lon0 = grid.axes['lon']['data'][0]
+        lat0 = grid.origin_latitude['data'][0]
+        lon0 = grid.origin_longitude['data'][0]
         self.proj = pyproj.Proj(proj='aeqd', datum='NAD83',
                                 lat_0=lat0, lon_0=lon0)
 
         # determine grid latitudes and longitudes.
-        x_1d = grid.axes['x_disp']['data']
-        y_1d = grid.axes['y_disp']['data']
+        x_1d = grid.regular_x['data']
+        y_1d = grid.regular_y['data']
         x_2d, y_2d = np.meshgrid(x_1d, y_1d)
```

Comment:
Again my plan was that this would be in another PR to fix up the **GridMapDisplay** class.  I was planning on using points_latitude and points_longitude here.